src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -410,9 +406,10 @@ export class AppComponent implements OnInit, AfterViewChecked {
                     self.placeHolderDataSets.push(undefinedDataSet);
                     self.messagesAdded = true;
                     self.onScroll(0);
-                }
+                break;
+                default:
```

Comment:
Might be useful for debugging to added a console.error message here saying there was an unexpect event emitted or something.